source/module_base/test/CMakeLists.txt

```diff
@@ -9,11 +9,12 @@ AddTest(
   TARGET base_atom_in
   SOURCES atom_in_test.cpp
 )
-AddTest(
-  TARGET base_blacs_connector
-  LIBS ${math_libs}
-  SOURCES blacs_connector_test.cpp
-)
+# Parallel test related to MPI should not be included in the base test set
```

Comment:
delete useless part.

Reply:
I will recover the original code and raise a new issue for this UT.